### DISCUSSION TITLE
Splash of color

### DISCUSSION THREAD

**Discussion by Beakfx-BPR** [jrowellfx/lsseq#15] [Ideas]
First off, brilliant piece of work, plain old ls has been missing this forever. I've had a mind (since 2003) to try to make something like this, as most linux tools fall short. I'm glad you did it!
   Perhaps I'm missing something, but I would love to see color coded output. Something like green for complete sequences, red for broken ones (like a frame or two stuck on the farm :) ), orange for movie files (the prores of you seq that your going to post), yellow, or cyan maybe? for solo image files, and white for non-media.
      If there is a way to do this already, could you point me to the relevant docs? Otherwise, such an implimentation would just make the whole thing more badass.

**Comment by jrowellfx** [comment]
Thanks for the compliment, I'm glad you like the work!

There currently is no way to do color-coded output as you've suggested. It feels like something that could be done with a separate command that you could pipe the output of lsseq through to give you what you want.

Note that the error-frame output (like missing or "bad" frames) already tells you info about whether a sequence is complete or not. That could also be used to help such a "filter" command like I suggested above.

I personally never use color coded output in my shell, as I find it distracting so unfortunately I'm not the best person to figure out the right approach. Sounds like one would have to follow the conventions that all other color-coded commands use, so that it's not an oddity for folks who do use color in their command-line usage. Sounds like a pretty big rabbit hole! Not that it isn't worth doing, it may be a nice addition.

You are the second person to ask for such a feature. So I'll start to think about  it, but in the short term, I do think you could get what you want by writing your own tool that you can pipe lsseq output through.  Yes? Perhaps?

Thanks again for the kind words.
